wiki/Visual_Settings/en.md

```diff
@@ -1,9 +1,12 @@
 # Visual Settings
 
-![Example of Visual Settings](img/VisualSettings.jpg "Example of Visual Settings")
+![Example of Visual Settings](./VisualSettings.jpg "Example of Visual Settings")
 
--   Visual Settings is below the screen, a few seconds before the beatmap starts. Having it open delays the beatmap from starting until your mouse pointer leaves it.
-    -   You can pause the game to open Visual Settings (Does not work on [Multi](/wiki/Multi)).
+**Visual Settings** is a hidden customisation overlay located at the bottom of the playfield. It can be accessed during the first few seconds before the beatmap starts. When the `Visual Settings` is shown, the beatmap will be delayed until the osu!cursor leaves the `Visual Settings` screen.
+
+Alternatively, it is possible to access `Visual Settings` by pausing the game. However, the pausing method will not work in a [Multi](/wiki/Multi "Multi") match due to the pause keybind being rebinded to an attempt to quitting the match and room instead.
+
+Do note that the setting changes from `Visual Settings` are by per beatmap and will be lost when exiting osu!. Global settings of the beatmap can be done in the [Options](/wiki/Options "Options") sidebar.
```

Comment:
*Note:* Setting changes from `Visual Settings` are saved per beatmap and will be lost after closing osu!. Global settings can be set in the Options sidebar.